README.md

```diff
@@ -7,8 +7,20 @@ The Yellow Paper is a formal definition of the Ethereum protocol, originally by
 
 It is free culture work, licensed under the Creative Commons Attribution Share-Alike (CC-BY-SA) version 4.0.
 
-## Usage
+## Releases
+This paper currently does not include changes to Ethereum after Byzantium, however if you like, feel free to make a PR to the [`byzantium` branch](https://github.com/ethereum/yellowpaper/tree/byzantium)! For further information on past hard forks, see [here](https://github.com/ethereum/EIPs/#past-hard-forks) as well as [here](https://github.com/ethereum/wiki/wiki/Releases) for previous and planned releases.
 
-The paper comes as a single ``latex`` file ``Paper.tex``. The latest version is generally available as a PDF at https://ethereum.github.io/yellowpaper/paper.pdf or just [yellowpaper.io](http://yellowpaper.io/) .
+## The paper
+The latest version is generally available as a PDF at https://ethereum.github.io/yellowpaper/paper. This version of the yellow paper can be used to get links that direct to specific info in the yellow paper, which is useful e.g. for pedagogical and academic purposes. [Here](
```

Comment:
This doesn't link to the PDF anymore.

Reply:
Hmm, it works for me! OK I'll remove the tips.

Reply:
![screenshot from 2018-01-20 21-53-06](https://user-images.githubusercontent.com/16969914/35182618-91e2e67c-fe2c-11e7-96d0-d4e14c46a41a.png)

Tips moved to #433.

Reply:
The link works in a Private tab in Brave, while not signed in to Github.

Reply:
https://github.com/ethereum/yellowpaper/files/1596574/Paper.pdf#blockhash

Reply:
Ah I think I know what you mean, the paper has been updated. I will revise.